Gems/GraphModel/Code/Include/GraphModel/Model/GraphContext.h

```diff
@@ -26,8 +26,13 @@ namespace GraphModel
     class GraphContext : public AZStd::enable_shared_from_this<GraphContext>
     {
     public:
+        AZ_CLASS_ALLOCATOR(GraphContext, AZ::SystemAllocator);
+        AZ_RTTI(GraphContext, "{4CD3C171-A7AA-4B62-96BB-F09F398A73E7}");
```

Comment:
The RTTI and CLASS_ALLOCATOR can be defined in the cpp file
```suggestion
        AZ_CLASS_ALLOCATOR_DECL
        AZ_TYPE_INFO_WITH_NAME_DECL(GraphContext);
        AZ_RTTI_NO_TYPE_INFO_DECL();
```

In the cpp file you could add
```
AZ_CLASS_ALLOCATOR_IMPL(GraphContext, AZ::SystemAllocator);
AZ_TYPE_INFO_WITH_NAME_IMPL(GraphContext, "GraphContext", "{4CD3C171-A7AA-4B62-96BB-F09F398A73E7}");
AZ_RTTI_NO_TYPE_INFO_IMPL(GraphContext);
```

Reply:
Agreed, now that these optimizations are in place. But this can be done in a separate PR. There are 32 instances of the class allocator macro in graph model. I have not searched for RTTI. There are also other gems that need to be done. They can be done in dedicated PRs, in batches, to make sure that nothing is missed and there are no mistakes.